HW_1/task_4.py

```diff
@@ -0,0 +1,12 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое количество
+# журавликов, а Катя сделала в два раза больше журавликов, чем Петя и
+# Сережа вместе?
+
+n = int(input("Введите количество журавликов: "))
+
+if n % 6 == 0:
+     print(n // 6, n * 2 // 3, n // 6)
```

Comment:
Проблема  с отступом
```suggestion
    print(n // 6, n * 2 // 3, n // 6)
```

Reply:
Кроме того, у вас тут дважды повторяется операция деления на 6. Но зачем это нужно, когда можно посчитать всего единожды и немного изменить второй аргумент
```suggestion
    res = n // 6
    print(res, res * 4, res)
```